src/browser/cacheDetail.js

```diff
@@ -26,6 +45,24 @@ function CacheDetailController($scope, SimDriver) {
         ctrl.cacheInfo = data;
     });
 
+    $scope.$on('updateModals', function (event, data) {
+        for (action in data) {
+            let cacheLevel = action.level;
+            let way = action.way;
```

Comment:
since this is the first time this has been referred to as `way` could you call it `assoc` or `associativity` so its consistent with the rest of the code.

Reply:
would associativity-way work?  something to specify there are multiple of them?  associativity is used elsewhere but in a different context

Reply:
maybe `num-sets`. I think my problem is that I didn't know what it meant without asking so the same could be said for others, and we want intuitive code. But that could have also just been me that was clue-less

Reply:
While it may not be intuitive, this is the correct name for a specific associativity column.  Ex: 4-way set associative cache